docker-compose-mongo.yml

```diff
@@ -0,0 +1,83 @@
+version: '3'
```

Comment:
Let's add a description of this yml to the README. We mention it briefly in the "quick run" section but let's call it out for what it does functionally - it spins up zookeeper/kafka a mongodb container, and kafka connect. Unlike the other docker-compose files, this one does NOT include the s3 depositor module.